templates/touch_sensor.service

```diff
@@ -4,7 +4,6 @@ Description=Monitor and restart touch_sensor.py
 [Service]
 Type=simple
 ExecStart=${PROJECT_PATH}/venv/bin/python3 ${PROJECT_PATH}/src/touch_sensor/touch_sensor.py
-User=root
 WorkingDirectory=${PROJECT_PATH}
 PIDFile=/var/run/touch_sensor.pid
 ExecStop=/bin/kill -15 \${MAINPID}
```

Comment:
Quotes around `${MAINPID}`

Reply:
This is actually not standard, at least in systemd service files as the environment variable expansion is handled by systemd.